gc/base/Configuration.cpp

```diff
@@ -169,9 +169,11 @@ MM_Configuration::initializeEnvironment(MM_EnvironmentBase* env)
 	case gc_modron_allocation_type_tlh:
```

Comment:
There is a TLH build flag as well.  Since you are cleaning up this function can you also wrap this under the appropriate #if defined as well?  Thanks